.github/workflows/post_to_mastodon.yml

```
@@ -0,0 +1,17 @@
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  post_to_mastodon:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post to Mastodon
+        run: |
```

actionlint

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)